src/DeprecatedHelpers/MyCollection/DeleteCollectedArtwork.ts

```diff
@@ -20,12 +24,14 @@ export interface DeleteCollectedArtworkArgs {
 export const deleteCollectedArtwork = ({
   contextOwnerId,
   contextOwnerSlug,
+  platform,
 }: DeleteCollectedArtworkArgs): DeleteCollectedArtwork => {
   return {
     action: ActionType.deleteCollectedArtwork,
     context_module: ContextModule.myCollectionArtwork,
     context_owner_id: contextOwnerId,
     context_owner_slug: contextOwnerSlug,
     context_owner_type: OwnerType.myCollectionArtwork,
+    platform,
```

Comment:
same as the above

Reply:
same comment as for https://github.com/artsy/cohesion/pull/354/commits/d57efdafc8f771e30a10ff2f0f897faad4afc613#r971891670

Reply:
let's delete both actions from Eigen and only use the interface `DeleteCollectedArtwork` or `AddCollectedArtwork`